core/src/main/java/org/frankframework/http/mime/MultipartEntityBuilder.java

```diff
@@ -58,7 +56,7 @@ public class MultipartEntityBuilder {
 	private boolean mtom = false;
 	private String firstPart = null;
 
-	private static final Random RANDOM = new Random();
+	private static final SecureRandom RANDOM = new SecureRandom();
```

Comment:
Hier heeft het geen toegevoegde waarde om hem secure te maken. Ik denk niet dat het (tegenwoordig) veel kwaad kan maar let er wel op, zomaar alles omzetten puur omdat Sonar dat zegt is ook niet de bedoeling.

Reply:
Eens, ik twijfelde ook bij deze en heb hem gewoon omgezet om vervolgens in de PR er over te hebben. De `SecureRandom` is zoals gezegd ook behoorlijk snel, dus veel verschil zal het niet maken. Maar goed, ik zet hem wel terug.